resources/views/login.blade.php

```diff
@@ -2,16 +2,17 @@
 <div class="container form">
         <div class="col-4">
             <h1>Vpiši se</h1>
-            <form method="post" action="register">
+            <form method="post" action="roomOption">
```

Comment:
This isn't good either. If this form handles login, then the action should be "login".